### PR TITLE
Move all the binaries and respective files into cmd/

### PR DESCRIPTION
**Pull request by mmlb**
Keeps all binaries and their support files grouped nicely instead of
littered randomly.

Signed-off-by: Manuel Mendez <manny@packet.com>

### CODE REVIEW

**Comment by gauravgahlot** [comment]
@mmlb Please resolve the conflicts.

**Comment by mmlb** [comment]
@gauravgahlot updated PTAL

File: cmd/tink-server/Dockerfile [hunk 1]
```diff
@@ -7,6 +7,6 @@ EXPOSE 42114
 RUN apk add --update ca-certificates && \
     apk add --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing cfssl
 
-COPY deploy/db/migrate /migrate
-COPY deploy/db/docker-entrypoint-initdb.d/tinkerbell-init.sql /init.sql
+COPY db/migrate /migrate
+COPY db/docker-entrypoint-initdb.d/tinkerbell-init.sql /init.sql
```
Comment (gauravgahlot) [inline cmd/tink-server/Dockerfile:11]: It should be `deploy/db/`.

**Comment by gauravgahlot** [comment]
@mmlb It would be great if we can conclude this PR asap.

**Comment by mmlb** [comment]
@gauravgahlot should be good now, PTAL.

**Comment by mmlb** [comment]
@gauravgahlot hmm that makes sense, I'll split the deploy files into those needed by tink-server and those for other things then.

**Pull request merged**